packages/react-core/src/demos/JumpLinks.md

```diff
@@ -38,6 +38,23 @@ ScrollspyH2 = () => {
   const headings = [1, 2, 3, 4, 5];
 
   const [isVertical, setIsVertical] = React.useState(true);
+  const [offsetHeight, setOffsetHeight] = React.useState(10);
+
+  // Update offset based on the masthead and jump links nav heights.
+  React.useEffect(() => {
+    const masthead = document.getElementsByClassName('pf-c-masthead')[0];
+    const offsetForPadding = 10;
+
+    getResizeObserver(masthead, () => {
```

Comment:
Im surprised this works with out explicitly importing `getResizeObserver` but I imagine products will need to explicitly import it, so may be worth adding.